tests/cpp/tensors/bfvtensor_test.cpp

```diff
@@ -0,0 +1,532 @@
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "tenseal/cpp/tenseal.h"
+
+namespace tenseal {
+namespace {
+
+using namespace ::testing;
+using namespace std;
+
+template <class Iterable>
+bool are_close(const Iterable& l, const std::vector<int64_t>& r) {
```

Comment:
`are_close` is relevant only for CKKS. For BFV, you should get the exact result. See the BFVVector C++ tests for reference

Reply:
Right thank you, I changed it now to assert/expect the exact result indeed